packages/editor/src/client/panel/selectEventsPanel.tsx

```diff
@@ -0,0 +1,199 @@
+import {ApolloError} from '@apollo/client'
```

Comment:
Copied from CommentBlock and adapted for EventBlock

Reply:
idem. copy code is evil ;)

Reply:
Tbf this one probably should be copied as you said we should show a proper preview in the future (same for comments). The other one I see what I can do D: